cmd/lansible

```diff
@@ -314,9 +314,16 @@ EOF
   elif [[ "$@" == *--playbook* ]] || [[ "$@" == *--with-playbook* ]]; then
     strap::bot "Run Ansible Playbook"
     strap::ansible::playbook::run "$@"
-  elif [[ -d "${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default" ]]; then
-    strap::bot "Run Ansible Playbook"
-    strap::ansible::playbook::run --with-playbook="${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default" "$@" 
+  elif [[ -z "$@" ]]; then
+    if [[ -f "${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml" ]]; then
+      strap::bot "Run Ansible Playbook"
+      strap::ansible::playbook::run --with-playbook="${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml" "$@"
```

Comment:
We shouldn't repeat full paths like this - let's create a variable to hold the file path that can be used in multiple locations, including the error messages.

Reply:
Sure.